packages/npm/@amazeelabs/scaffold/README.md

```diff
@@ -0,0 +1,24 @@
+# Amazee Package Scaffolder
+
+Scaffold code packages. Automatically adds and configures code quality and
+testing tools.
+
+## Installation
+
+```shell
+yarn add @amazeelabs/scaffold
+yarn amazee-scaffold
+```
+
+This will alter the packages `.gitignore` file, add a `postinstall` hook and
+make sure that configurations for code quality tools like `eslint`, `prettier`
+or `jest` are in place. The scaffolded configuration files should are added to
+`.gitignore` and should not be committed. They will be auto-added and updated
+with every `yarn install`.
+
+## Usage
+
+After install there is a `test` npm script that will run tests either in CI-mode
+or in watch-mode, depending on the current environment. It will also look for an
+optional `test.sh` in the package root that will also be executed when running
+in an CI environment.
```

Comment:
`test.sh` is not very intuitive for me. Also I personally do not like when local and CI commands work too differently. But I always can run `CI=true yarn test`, so all good.
(Just thoughts 😄 )

Reply:
You can always run `yarn test:ci` too. And the "local" runs the "watchable" tests (jest, eslint).
But they diverged a lot, I see that. Maybe we even remove `is-ci-cli`, have a `test` command and another `watch` command to run during development?

Reply:
Yes, would be great!